packages/neos-ui-redux-store/src/CR/Nodes/selectors.js

```diff
@@ -205,19 +205,23 @@ export const getPathInNode = (state, contextPath, propertyPath) => {
 
 export const makeGetAllowedChildNodeTypesSelector = (nodeTypesRegistry, elevator = id => id) => createSelector(
     [
-        (state, {reference}) => getPathInNode(state, elevator(reference), 'policy.canEdit'),
         (state, {reference}) => getPathInNode(state, elevator(reference), 'policy.disallowedNodeTypes'),
         (state, {reference}) => getPathInNode(state, elevator(reference), 'isAutoCreated'),
         (state, {reference}) => getPathInNode(state, elevator(reference), 'name'),
         (state, {reference}) => getPathInNode(state, elevator(reference), 'nodeType'),
         (state, {reference}) => getPathInNode(state, elevator(parentNodeContextPath(reference)), 'nodeType'),
         (state, {role}) => role
     ],
-    (canEdit, disallowedNodeTypes, ...args) => canEdit ?
-        nodeTypesRegistry
+    (disallowedNodeTypes, ...args) => {
+        let allowedNodeTypesTakingAutoCreatedIntoAccount = nodeTypesRegistry
```

Comment:
use `const` because allowedNodeTypesTakingAutoCreatedIntoAccount is never reassigned